packages/teams-js/src/internal/communication.ts

```diff
@@ -706,6 +711,19 @@ function handleParentMessage(evt: DOMMessageEvent): void {
     if (callbackId) {
       const callback = CommunicationPrivate.callbacks.get(callbackId);
       logger('Received a response from parent for message %i', callbackId);
+
+      // Send performance metrics information of message delay
+      const callbackInformation = CommunicationPrivate.callbackInformation.get(callbackId);
+      if (callbackInformation && message.timestamp) {
+        handleHubToAppPerformanceMetrics({
+          actionName: callbackInformation.name,
+          messageDelay: getCurrentTimestamp() - message.timestamp,
```

Comment:
I haven't taken a look at hub sdk side's of code, but just reading codes here, it looks this messageDelay measures between "when app sdk receiving response" and "when hub sdk sends over the response". That sides makes sense. 

What about the other side? i.e, messageDelay between "when app sdk sends the request to hub sdk" and "when hub sdk receives the message request from app sdk"? I do see `messageWasCreatedAt`(the time that app sdk sends the request to hub sdk), do we also need to know "the time when hub sdk receives the message request" to calculate the other side of the latency?